### PR TITLE
feat(ui/primitives): command component

### LINKED ISSUE #360: [Component]: Command component
### Purpose?

A component which adds command styling to its children. In the future, we can add a copy button as well.

### Concepts

N/A

### Description

Example code usage:

```tsx
<Command>/channel-settings</Command>
```

### Extra Information

_No response_

### PR DESCRIPTION
**Pull request by JoltCode**
# Description

Slash command component

Fixes #360 (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

N/A

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works 
- [x] I have updated .env.example if I added a new environment variable
- [x] My PR title follows the [semantic commits](https://www.conventionalcommits.org/en/v1.0.0/) style

### CODE REVIEW

File: packages/ui/src/components/primitives/base/Icons.tsx [hunk 1]
```diff
@@ -139,15 +139,15 @@ export const ExternalLinkIcon = () => {
 	);
 };
 
-export const CopyIcon = () => {
+export const CopyIcon = ({ className }: { className?: string }) => {
```
Comment (RhysSullivan) [inline packages/ui/src/components/primitives/base/Icons.tsx:142]: Headless icons probably has a copy icon we can use instead of duplicating it here

**Review by RhysSullivan — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/primitives/base/Command.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}> = ({ command }) => {
+	const [, copy] = useCopyToClipboard();
+
+	return (
+		<div className="flex flex-col">
+			<div className="flex w-max flex-row items-center justify-center gap-5 rounded-standard border-1 border-[#CBCED1] bg-[#F4F4F4] px-4 py-2 text-black dark:border-[#32353A] dark:bg-[#1F2226] dark:text-white">
+				<span className="font-body text-lg">/{command}</span>
+				<Button
+					className="h-auto rounded-standard border-2 border-[#d4d4d4] bg-[#F2F2F2] p-1 transition-colors hover:bg-[#e6e6e6] dark:border-0 dark:bg-[#2A2E33] dark:drop-shadow-2xl dark:hover:bg-[#363b42]"
+					onClick={() => copy(`/${command}`)}
```
Comment (RhysSullivan) [inline packages/ui/src/components/primitives/base/Command.tsx:21]: Needs a pop up toast that says "copied!" On click

**Review by RhysSullivan — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/primitives/base/Command.tsx (hunk 2 above)
Comment (JoltCode) [reply packages/ui/src/components/primitives/base/Command.tsx:21]: Done!

File: packages/ui/src/components/primitives/base/Command.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	 */
+	command: string;
+}> = ({ command }) => {
+	const [, copy] = useCopyToClipboard();
+
+	return (
+		<div className="flex flex-col">
+			<div className="flex w-max flex-row items-center justify-center gap-5 rounded-standard border-1 border-[#CBCED1] bg-[#F4F4F4] px-4 py-2 text-black dark:border-[#32353A] dark:bg-[#1F2226] dark:text-white">
+				<span className="font-body text-lg">/{command}</span>
+				<Button
```
[the remark this answers is not in the document]
Comment (JoltCode) [reply packages/ui/src/components/primitives/base/Command.tsx:19]: Done :+1:

File: packages/ui/src/components/primitives/base/Icons.tsx (hunk 1 above)
Comment (JoltCode) [reply packages/ui/src/components/primitives/base/Icons.tsx:142]: :+1:

**Pull request merged**